src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
Instead of creating multiple instances of 'Lottery', consider reusing a single instance since it can fulfill the task of generating multiple 'Ball' objects.

Reply:
Fixed